### DISCUSSION TITLE
Display more features

### DISCUSSION THREAD

**Discussion by jeroenst** [benzino77/tasmocompiler#294] [Q&A] [answered]
In tasmocompiler it's possible to select just a few features, the other features has to be specified by defining defines.

Is it possible to add more features (like but not only serial bridge or modbus bridge which is in the development branch)?

**Comment by benzino77** [comment, marked as answer]
Sure it is possible to add more features. Pull Requests are welcome!!!

**Reply by Mark-Muc** [reply]
shutter support will be often needed
incl. MOTOR_STOP_TIME = xxx
xxx in ms ; default 500
Mark

#ifndef USE_SHUTTER
#define USE_SHUTTER     
#endif

#define MOTOR_STOP_TIME 750

**Reply by benzino77** [reply]
I've added "Shutters and Blinds" to the list of available features. It is for now in development branch.
After choosing this feature you would have an option to change:
```C
SHUTTER_RELAY_OPERATION_TIME
MOTOR_STOP_TIME
HUTTER_CLEAR_PWM_ONSTOP
```
parameters via "Custom parameters" field.

**Reply by Mark-Muc** [reply]
well done!!
Thank you .. I think it will help a lot of people!!
Mark

**Reply by Mark-Muc** [reply]
i am right?
![image](https://user-images.githubusercontent.com/67410847/195975059-b7d9ffcf-97fe-4056-ba98-0b4ad7c6085f.png)

but can't see?
Thx
Mark

**Reply by benzino77** [reply]
You have to start new workspace on gitpod. The easiest way is to click on this [link](https://gitpod.io/#https://github.com/benzino77/tasmocompiler/tree/development)

![Screenshot from 2022-10-15 11-05-22](https://user-images.githubusercontent.com/15658235/195978650-18468e93-b27c-41ba-8c30-b821fbd9a13e.png)

**Comment by Mark-Muc** [comment]
many THX .. IT WORKED!!
Regards Mark

**Comment by jeroenst** [comment]
I also added Dutch language and ModbusBridge.

**Comment by Massimo65** [comment]
Good morning
I just started using this fantastic software, it will be possible for "shelly-type" hardware to choose between esp8266_2M1M (default) and esp8266_2M256 boards.
Thank you

**Reply by benzino77** [reply]
I don't want make UI overloaded with possible choices (it is already a little bit). But...

You can make fork of TasmoCompiler repo. Then, in your fork, change line 42 in file in file `src/components/AppStepper/FeaturesStep/AvailableBoards.js` to the value you need (`esp8266_2M256` in your case) then point gitpod to your fork or build your own docker image (Dockerfile comes with TasmoCompiler)
You can also add completely new board definition (have a look in `src/components/AppStepper/FeaturesStep/AvailableBoards.js` file)
